src/pages/contact_confirmation.js

```diff
@@ -0,0 +1,45 @@
+/** @jsx jsx */
+import { useState } from 'react';
+import { jsx } from '@emotion/core';
+import { useHistory } from 'react-router-dom';
+
+import { Routes } from 'constants/Routes';
+import { routeWithParams } from 'lib/utils/routes';
+
+import Page from 'components/layouts/Page';
+import PageLoader from 'components/Loader/PageLoader';
+import { ContactConfirmation } from 'components/Confirmation';
+
+const TEMP_NAME = 'Temp User';
+const TEMP_CONTACT = 'medicalWorker@healthcarefacility.com';
```

Comment:
Temp values while we build backend for this part.

Reply:
could you add a note above this?